src/ensembl/README.md

```diff
@@ -8,29 +8,35 @@ The tools that need setting up are Node.js and the IDE. To setup Node.js you onl
 
 There are several script commands that have been baked into the NPM configuration. They have been briefly described below:
 
-1. `npm start` - This should start the development server via `webpack`. There is no need to do any compilation manually, as `webpack` will take care of it.
-2. `npm run serve:dev` - This does the same as `npm start`. The latter is actually an alias to this script.
-3. `npm run build` - Runs the production build. It will initially delete the existing local production build and replace it with the new one.
-4. `npm run serve:prod` - This runs the built production site locally.
-5. `npm run lint:scripts` - Runs TSLint to check code errors in TypeScript and React. Also, suggests improvements to the code.
-6. `npm run lint:style` - Does the same as above through `stylelint` for the SASS code.
-7. `npm run lint` - Runs both `lint:scripts` and `lint:style`. There is no need to run any of these lint scripts during development. This is because, each time a change is made to the code, the linters run along with the build.
-8. `npm test` - Runs `jest` to check whether the unit tests pass.
-9. `npm run coverage` - Updates the `jest` coverage of the React.js code, and shows the test coverage.
-10. `npm run shanpshot` - Updates the `jest` snapshots. You will sometimes need to run this after a change is made to the React.js code. However, first check whether the unit tests are passing after updating them. If you get any errors related to `jest` snapshots, you will then need to run this script.
-11. `npm run storybook` - Starts the Storybook application
-12. `npm run deploy-storybook` — Builds the Storybook application and deploys it to Github Pages ([link](https://ensembl.github.io/ensembl-client))
-12. `npm run check-types` – Runs typescript compiler to check for type correctness
+- `npm run copy-dotenv` - Makes a copy of `.env.example` file named as `.env` in case it doesn't exist. This command will always run before `serve:dev`.
+- `npm start` - This should start the development server via `webpack`. There is no need to do any compilation manually, as `webpack` will take care of it.
+- `npm run serve:dev` - This does the same as `npm start`. The latter is actually an alias to this script.
+- `npm run serve:prod` - This runs the built production site locally.
```

Comment:
Do we need to run `npm run certify` before attempting to run prod? I couldn't make prod to work because it was complaining of missing certificates.

Reply:
I am undecided on whether we should allow the user/developer to run the production test server in his preferred protocol. This would get rid of the need to run the `certify` command before the production server runs.

Reply:
👌

_Decided to have another PR where we'll make sure that `npm run serve:prod` works on local machine even without issuing a certificate._